cmd/wallet/main.go

```diff
@@ -1,163 +1,391 @@
 package main
 
 import (
-	"errors"
+	"encoding/binary"
+	"flag"
 	"fmt"
 	"io/fs"
+	"log"
 	"os"
 	"os/user"
 	"path/filepath"
+	"strings"
 
-	"github.com/howeyc/gopass"
 	"github.com/mr-tron/base58"
-	flag "github.com/spf13/pflag"
+
+	"github.com/pkg/errors"
+
+	"github.com/howeyc/gopass"
+	"github.com/tyler-smith/go-bip39"
+	"github.com/wavesplatform/gowaves/pkg/crypto"
+	"github.com/wavesplatform/gowaves/pkg/proto"
 	"github.com/wavesplatform/gowaves/pkg/wallet"
 )
 
-var usage = `
+const (
+	newOpt               = "new"
+	showOpt              = "show"
+	seedPhraseOpt        = "seed-phrase"
+	seedPhraseBase58Opt  = "seed-phrase-base58"
+	accountSeedBase58Opt = "account-seed-base58"
+
+	schemeOpt = "scheme"
+)
 
+const (
+	defaultBitSize    = 160
+	walletDefaultName = ".waves"
+)
+
+var usage = `
 Usage:
-  wallet command [flags]
+  wallet [flags]
 
-Available Commands:
-  add          Add seed to wallet
-  show         Print wallet data
+Flags:
+`
 
+var examples = `
+Examples:
+	./wallet -show -wallet <wallet path> // Show existing wallet credentials
+	./wallet -new // Generate a seed phrase and a wallet 
+	./wallet -seed-phrase "..." // Import a seed phrase
+	./wallet -seed-phrase-base58 "..." // Import a Base58 encoded seed phrase
```

Comment:
From example `-seed-phrase-base58` looks like a string option, but actually it is boolean.

Reply:
Once again, it's supposed to be string